platform/security/sts/security-sts-propertyclaimshandler/src/test/java/org/codice/ddf/security/sts/claims/property/TestAttributeFileClaimsHandler.java

```diff
@@ -158,6 +162,47 @@ public void testRetrieveClaimsValuesNullPrincipal() {
     ProcessedClaimCollection processedClaims =
         attributeFileClaimsHandler.retrieveClaimValues(claimCollection, claimsParameters);
 
-    Assert.assertThat(processedClaims.size(), CoreMatchers.is(equalTo(0)));
+    assertThat(processedClaims, is(empty()));
+  }
+
+  @Test
+  public void testGetSingleValuedSystemHighAttribute() throws IOException {
+    assertThat(
+        attributeFileClaimsHandler.getValues("test"),
+        isPresentAnd(containsInAnyOrder("testValue")));
+  }
+
+  @Test
+  public void testGetMultiValuedSystemHighAttribute() throws IOException {
+    assertThat(
+        attributeFileClaimsHandler.getValues("test1"),
+        isPresentAnd(containsInAnyOrder("testing1", "testing2", "testing3")));
+  }
+
+  @Test
+  public void testNoMatchGetSystemHighAttribute() throws IOException {
+    System.setProperty(SystemBaseUrl.HOST, "admin");
```

Comment:
❓ Is this needed, or is it to make it more clear? It looks like it's already being set in the `@Before`.